.github/workflows/kind.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
           kubectl wait --for=condition=Ready pods --all -n kube-system
 
           # Deploy flotta operator
+          sed -i 's/LOG_LEVEL=info/LOG_LEVEL=debug/g' config/manager/controller_manager_config.properties
```

Comment:
We can use this method to change log verbosity (without changing the codebase for it):
https://github.com/project-flotta/flotta-operator/blob/main/docs/user-guide/logger.md#change-the-verbosity-of-the-logger
```
kubectl patch cm -n flotta flotta-operator-manager-config --type merge --patch '{"data":{"LOG_LEVEL": "debug"}}'
```
it can be added after `make depoy` so the next command to wait for flotta pods will be there.

Reply:
I had this doubt too, but I thought he wanted to avoid restarting the controller to speed up the kind task.

Reply:
Gloria you're  right!